snappy/install.go

```diff
@@ -45,13 +45,34 @@ const (
 	AllowGadget
 )
 
-func doUpdate(part Part, flags InstallFlags, meter progress.Meter) error {
-	if _, err := part.Install(meter, flags); err == ErrSideLoaded {
+func installRemote(mStore *SnapUbuntuStoreRepository, remoteSnap *RemoteSnapPart, flags InstallFlags, meter progress.Meter) (string, error) {
+	downloadedSnap, err := mStore.Download(remoteSnap, meter)
+	if err != nil {
+		return "", fmt.Errorf("can not download %s: %s", remoteSnap.Name(), err)
```

Comment:
We've actually been using "cannot", at least in the reviews I've been in. Seems like the better choice for being shorter and easier to read/write/say. If we change this, let's please sync on it so we're aligned.